plugins/flytekit-data-fsspec/flytekitplugins/fsspec/__init__.py

```diff
@@ -25,13 +25,15 @@
 import importlib
 
 from flytekit import StructuredDatasetTransformerEngine, logger
-from flytekit.configuration import internal
-from flytekit.types.structured.structured_dataset import ABFS, GCS, S3
 
 from .arrow import ArrowToParquetEncodingHandler, ParquetToArrowDecodingHandler
 from .pandas import PandasToParquetEncodingHandler, ParquetToPandasDecodingHandler
 from .persist import FSSpecPersistence
 
+S3 = "s3"
```

Comment:
We can import this from https://github.com/flyteorg/flytekit/blob/31277351c831b195951f1bc1152a4d08338e5633/flytekit/types/structured/structured_dataset.py#L41-L43

Reply:
oh crap, no i'm deleting these.

Reply:
structured dataset should not know about protocols at all really.  deleted.